csharp-basic-topics/UsingDirectiveForAdditionalTypes/UsingDirectiveForAdditionalTypes/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
This `using` directive looks like it can be removed.

Reply:
I understand that you mean using System.Collections.Generic; - correct!